src/screens/Home/Home.js

```diff
@@ -454,27 +448,31 @@ class HomeScreen extends React.Component<Props, State> {
               headerComponent={(
                 <React.Fragment>
                   <WalletsPart handleWalletChange={this.handleWalletChange} isChanging={!!loaderMessage} />
-                  <BadgesWrapper>
-                    <ListHeader>Game of badges</ListHeader>
-                    <FlatList
-                      data={badges}
-                      horizontal
-                      keyExtractor={(item) => (item.id.toString())}
-                      renderItem={this.renderBadge}
-                      style={{ width: '100%', paddingBottom: spacing.medium }}
-                      contentContainerStyle={{ paddingHorizontal: 6, ...badgesContainerStyle }}
-                      initialNumToRender={5}
-                      ListEmptyComponent={(
-                        <EmptyStateWrapper>
-                          <EmptyStateParagraph
-                            title="No badges"
-                            bodyText="You do not have badges yet"
-                          />
-                        </EmptyStateWrapper>
-                    )}
-                    />
-                  </BadgesWrapper>
-                  {!!referralsFeatureEnabled && this.renderReferral(colors)}
+                  {!!referralsFeatureEnabled && this.renderReferral()}
+                  <CollapsibleSection
+                    label="Game of badges"
+                    collapseContent={
+                      <FlatList
+                        data={badges}
+                        horizontal
+                        keyExtractor={(item) => (item.id.toString())}
+                        renderItem={this.renderBadge}
+                        style={{ width: '100%', paddingBottom: spacing.medium }}
+                        contentContainerStyle={{ paddingHorizontal: 2, paddingTop: 16, ...badgesContainerStyle }}
+                        initialNumToRender={5}
+                        ListEmptyComponent={(
+                          <EmptyStateWrapper>
+                            <EmptyStateParagraph
+                              title="No badges"
+                              bodyText="You do not have badges yet"
+                            />
+                          </EmptyStateWrapper>
+                        )}
+                      />
+                    }
+                    onPress={() => { this.setState({ isBadgesCollapsed: !isBadgesCollapsed }); }}
```

Comment:
Maybe we should store that setting in storage?

Reply:
Good call - will update. Thanks :)